.github/workflows/library_net_tests.yml

```diff
@@ -115,77 +117,78 @@ jobs:
               --framework net6.0
           fi
 
-      - name: Fetch awses-decrypt/python-2.3.0.zip
-        working-directory: ./
-        shell: bash
-        run: |
-          PYTHON_23_VECTOR_PATH=$GITHUB_WORKSPACE/python23/vectors
-          mkdir -p $PYTHON_23_VECTOR_PATH
-          DOWNLOAD_NAME=python23.zip
-          curl --no-progress-meter --output $DOWNLOAD_NAME --location $VECTORS_URL
-          unzip -o -qq $DOWNLOAD_NAME  -d $PYTHON_23_VECTOR_PATH
-          rm  $DOWNLOAD_NAME
-
-      - name: Run Test Vectors on .NET Framework net48
-        working-directory: ./AwsEncryptionSDK/runtimes/net/TestVectorsNative/TestVectors
-        shell: bash
-        run: |
-          PYTHON_23_VECTOR_PATH=$GITHUB_WORKSPACE/python23/vectors
-          DAFNY_AWS_ESDK_TEST_VECTOR_MANIFEST_PATH="$PYTHON_23_VECTOR_PATH/manifest.json" \
-          dotnet test --framework net48
+      # - name: Fetch awses-decrypt/python-2.3.0.zip
+      #   working-directory: ./
+      #   shell: bash
+      #   run: |
+      #     PYTHON_23_VECTOR_PATH=$GITHUB_WORKSPACE/python23/vectors
+      #     mkdir -p $PYTHON_23_VECTOR_PATH
+      #     DOWNLOAD_NAME=python23.zip
+      #     curl --no-progress-meter --output $DOWNLOAD_NAME --location $VECTORS_URL
+      #     unzip -o -qq $DOWNLOAD_NAME  -d $PYTHON_23_VECTOR_PATH
+      #     rm  $DOWNLOAD_NAME
```

Comment:
Question/Issue: Why not just delete this content?

Reply:
testing stuff out, not want to delete this yet